cypress/pageobject/Facility/FacilityHome.ts

```diff
@@ -104,6 +107,33 @@ class FacilityHome {
     const encodedText = encodeURIComponent(searchText).replace(/%20/g, "+");
     this.getURL().should("include", `search=${encodedText}`);
   }
+
+  toggleSidebar() {
+    this.sidebarToggle().should("be.visible").click();
+  }
+
+  private verifySidebarElements(
+    textVisibility: "be.visible" | "not.be.visible",
+  ) {
+    this.sidebarItems()
+      .should("have.length.at.least", 1)
+      .each(($item) => {
+        cy.wrap($item, { timeout: 10000 })
+          .find('[data-testid="sidebar-icon"]')
+          .should("be.visible");
+        cy.wrap($item, { timeout: 10000 })
+          .find('[data-testid="sidebar-text"]')
+          .should(textVisibility);
```

Comment:
- verify the text itself, not just random text is visible , verify for the facility icon the corresponding text is "Facilities" itself, similar for other icons as well